packages/stark-ui/src/modules/minimap/components/minimap.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="stark-minimap">
+	<div class="stark-minimap-dots">
+		<ul>
+			<li *ngFor="let item of items; trackBy: $index"
```

Comment:
Currently, Travis is failing because of this: `$index`.... I think you should put a function here instead